src/layers/legacy/medImageIO/CMakeLists.txt

```diff
@@ -65,35 +74,12 @@ target_include_directories(${TARGET_NAME}
 target_link_libraries(${TARGET_NAME}
   Qt5::Core
   Qt5::Widgets
+  ${ITK_LIBRARIES}
+  ${VTK_LIBRARIES}
   dtkCoreSupport
   dtkLog
   medCore
   medCoreLegacy
-  ITKCommon
```

Comment:
i'm curious, why do you prefer them in the find_package than here?

Reply:
I like to put them in the find required section. And I hope that one time I could remove most of them .